### PR TITLE
Switched to new Algolia account

### PR DESCRIPTION
**Pull request by jroper**
Algolia is changing the way they do DocSearch, instead of it being something that they manage and we have to email them to get configuration changes, they now give us free Algolia accounts so we can configure it ourselves. I've updated the search to use the new account.  Currently, only I have access to this account I think, but I guess I can add other people.

I also fixed a bug - it was only searching the current docs, this change means it now searches the version that the user is currently viewing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,15 @@ The documentation is generated from the lagom main project as unstyled HTML snip
...
 ## Developing the website
 
+Due to bintray being shutdown, some of the dependencies of this build are not available for download from anywhere. The easiest way to address this is build them locally:
+
+```
+git clone https://github.com/lightbend/lightbend-markdown.git
+cd lightbend-markdown
+git checkout v1.7.0
+sbt publishLocal
+```
```
Comment (jroper) [inline README.md:33]: @octonato I noticed you've built/deployed the docs yourself, is this what you did to get this dependency?

File: build.sbt [hunk 2]
```diff
@@ -20,8 +20,6 @@ libraryDependencies ++= Seq(
   "org.yaml" % "snakeyaml" % "1.12"
 )
 
-resolvers += Resolver.bintrayIvyRepo("typesafe", "ivy-releases")
-
```
Comment (jroper) [inline build.sbt:24]: Removed because bintray no longer exists.

File: src/main/twirl/com/lightbend/lagom/docs/documentation.scala.html [hunk 3]
```diff
@@ -81,7 +81,7 @@
 
             };
 
-            var lagomFacetFilters = ['version:current'];
+            var lagomFacetFilters = ['version:@lang.name'];
```
Comment (jroper) [inline src/main/twirl/com/lightbend/lagom/docs/documentation.scala.html:84]: I wonder how long this bug existed for!

**Comment by jroper** [comment]
@octonato I just added you to the Algolia lagomframework search team.

File: README.md (hunk 1 above)
Comment (octonato) [reply README.md:33]: I didn't build the docs, but only updated the release page. 

For the record, I did so because Vegemite is gone and I had to publish from my machine.

**Review by octonato — APPROVED** [review]
LGTM

File: src/main/twirl/com/lightbend/lagom/docs/documentation.scala.html (hunk 3 above)
Comment (ignasi35) [reply src/main/twirl/com/lightbend/lagom/docs/documentation.scala.html:84]: How does this work?

Won't this try to search by `version:scala`  instead of `version:1.6.x`?

Long ago, when extending search to be cross-project, the agreement was that all searches would be on `current`  as a way to push users towards latest. So, hardcoding `version:current` was not a bug but a feature.

Comment (jroper) [reply src/main/twirl/com/lightbend/lagom/docs/documentation.scala.html:84]: `lang` is an instance of `LanguageVersion`, the `name` field of `LanguageVersion` contains the name of the version, not the name of the language.

That seems really odd to me re:hardcoding current... if I'm in the 1.5.x docs, because I'm using Lagom 1.5.x, then when I search, I want to get 1.5.x results. I don't see how returning current results helps anyone, that would frustrate me as a user. I don't think anyone ever says "we should upgrade to the latest version because when we search in the docs we get silently switched to the latest version in the docs", that just would never appear as a reason. If it's a reason for any decisions being made, it might be a reason why people might stop using Lagom... "the docs are really hard to use because when you search you get switched to the latest version".

Comment (ignasi35) [reply src/main/twirl/com/lightbend/lagom/docs/documentation.scala.html:84]: I'm sure the people that made that decision understand the trade-offs and still preferred `current`.

Let me rephrase: it's not a bug it's an opinion.
